Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py

```diff
@@ -149,27 +148,33 @@ def get_reputation_score(indicator_types: list[str]) -> int:
     return Common.DBotScore.NONE
 
 
-def extract_indicator_from_pattern(pattern_str: str) -> str | None:
+def extract_indicator_from_pattern(stix_object: dict) -> str:
     """
     Extract indicator from STIX indicator pattern.
     i.e.
         [ipv4-addr:value = '198.51.100.1/32'] => 198.51.100.1/32
         [ipv4-addr:value = '198.51.100.1/32' OR ipv4-addr:value = '203.0.113.33/32'] => 198.51.100.1/32
     """
-    pattern = PatternParser(pattern_str)
+    if 'pattern' not in stix_object:
+        return stix_object['name']
```

Comment:
When you run a reputation command, it calls:
- `get_indicator_context_command()` which:
1. fetches the indicator from Sekoia.io's API https://github.com/SEKOIA-IO/Cortex-XSOAR-integration/blob/d80adc591e5735aca1d5804b55fd96dc6060393d/Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py#L654
2. `extract_indicators()` from API response https://github.com/SEKOIA-IO/Cortex-XSOAR-integration/blob/d80adc591e5735aca1d5804b55fd96dc6060393d/Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py#L664
- when `extract_indicators()` is called:
1. it sets a default value for name and type to the STIX object if empty https://github.com/SEKOIA-IO/Cortex-XSOAR-integration/blob/d80adc591e5735aca1d5804b55fd96dc6060393d/Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py#L471
- `get_stix_object_reputation()` is called for the empty stix object which is in charge of calling a method according to the indicator type
- the choose method will then call `extract_indicator_from_pattern()` which will answer by `stix_object['name']` because the indicator is unknown

I have chosen to add support of unknown indicator this way because all the `CommandResults` is already complete. Which means that I don't have to duplicate the code for unknown indicator, only add support.